### DISCUSSION TITLE
tailwind / problem with opacity althought using absolute

### DISCUSSION THREAD

**Discussion by robokonk** [tailwindlabs/tailwindcss#9103] [Help]
I have problem with remove opacity with my modal:
 - code (line 174): https://github.com/robokonk/tailwind-opacity/blob/b407b9c1c9e0787c241b33086c60ba6a579a0ca1/src/components/Header.tsx#L174
 - live version:https://melodic-florentine-ff2a5b.netlify.app/
 - image (problems): https://i.imgur.com/Fovposz.png

How I can remove from this modal opacity? And why actuall method is not working? I added absolute and z-...?

**Comment by wongjn** [comment]
The whole thing is slightly translucent since a parent element has an opacity class:

```html
<div class="… bg-gray-700 opacity-75 …">
  <!-- … -->
</div>
```

What you may be intending is to make the background color have opacity, but the content to be fully opaque:

```diff
 <div
   className={`${
     open
-       ? "fixed top-0 left-0 right-0 bottom-0 w-full h-screen z-50 overflow-hidden bg-gray-700 opacity-75 flex flex-col items-center justify-center"
+       ? "fixed top-0 left-0 right-0 bottom-0 w-full h-screen z-50 overflow-hidden bg-gray-700/75 flex flex-col items-center justify-center"
       : ""
   }`}
 >
```
